packages/bpk-component-bottom-sheet/src/BpkBottomSheet.module.scss

```diff
@@ -17,16 +17,156 @@
+    transform tokens.$bpk-duration-sm ease-in-out;
+  outline: 0;
+  background-color: tokens.$bpk-modal-background-color;
+  opacity: tokens.$bpk-modal-opacity;
+
+  // required to prevent the header extending beyond the rounded corners
+  overflow: hidden;
+  overflow-y: scroll;
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+
+  // FIX: Prevents ugly flickering when tapping inside the modal in Mobile Safari
+  -webkit-tap-highlight-color: transparent;
+
+  @include shadows.bpk-box-shadow-xl;
+  @include radii.bpk-border-radius-sm;
+
+  @include breakpoints.bpk-breakpoint-mobile {
+    position: fixed;
+    bottom: 0;
+    height: fit-content;
+    max-height: 90%;
+    margin-bottom: 0;
+    border-radius: tokens.$bpk-border-radius-lg tokens.$bpk-border-radius-lg 0 0;
+    overflow-x: hidden;
+  }
+
+  &::-webkit-scrollbar {
+    display: none;
+  }
+
+  &--appear {
+    animation-duration: 0.24s;
+    animation-name: slide-up;
+    animation-timing-function: ease-in-out;
+
+    @include breakpoints.bpk-breakpoint-above-mobile {
+      transform: scale(0.9);
+      opacity: tokens.$bpk-modal-initial-opacity;
+      animation: none;
+    }
+  }
+
+  &--appear-active {
+    @include breakpoints.bpk-breakpoint-above-mobile {
+      transform: scale(1);
+      opacity: tokens.$bpk-modal-opacity;
+    }
+  }
+
+  &--exit {
+    animation-direction: reverse;
+    animation-duration: 0.24s;
+    animation-name: slide-up;
+    animation-timing-function: ease-in-out;
+
+    @include breakpoints.bpk-breakpoint-above-mobile {
+      animation: none;
+    }
+  }
+
+  &--content {
+    padding: tokens.$bpk-modal-content-padding;
+    flex: 1;
+    overflow-y: auto;
+  }
+
+  &--wide {
+    @include breakpoints.bpk-breakpoint-above-mobile {
+      max-width: tokens.$bpk-modal-wide-max-width;
+    }
+  }
+
+  &--header {
+    position: sticky;
+    top: 0;
+    z-index: tokens.$bpk-zindex-tooltip - 1;
+
+    @include borders.bpk-border-bottom-sm(tokens.$bpk-line-day);
+  }
+
+  &--navigation {
     display: flex;
-    padding: tokens.bpk-spacing-lg();
+    justify-content: space-between;
+    background-color: tokens.$bpk-modal-background-color;
+  }
+
+  &--heading {
+    width: calc(
+      100% - ((#{$close-button-width}) * 2 + #{tokens.bpk-spacing-sm()} / 2)
+    ); // Treat title as if it was always surrounded by close and action to center align it
+
+    margin-right: auto;
+    text-align: center;
+
+    @include typography.bpk-text;
+    @include typography.bpk-label-1;
+  }
+
+  &--close-button {
+    position: relative; // Override position: absolute coming from BpkNavigationBar
+    left: auto;
+    margin-right: tokens.bpk-spacing-lg();
+
+    @include utils.bpk-rtl {
+      right: auto;
+      margin-right: calc(-#{tokens.bpk-spacing-sm()} / 2);
+      margin-left: tokens.bpk-spacing-lg();
+    }
+  }
+
+  &--action-button {
+    position: relative; // Override position: absolute coming from BpkNavigationBar
```

Comment:
Waht would be the reason for overriding BpkNavigationBar?

This could indicate something is wrong in the core component

Reply:
The X button and action button overlays each other as out of the box, the `BpkNavigationBar` has a styling for the leading item to have absolute positioning (see the screenshot) which causes the buttons to overlap.
<img width="540" alt="image" src="https://github.com/Skyscanner/backpack/assets/41451092/7845ae32-7165-4364-8644-95107210fbd9">

Reply:
Is this a symptom of other styles we are also supplying to parts of the Nav bar as out of the box this should be supporting and aligning all the parts equally to make this possible without the need for overridding as otherwise we are introducing a risk of CSS non determinism and this problem coming up anway even with the overrides

Reply:
It doesn't appear so, as removing all styling relating to components at or within `BpkNavigationBar` is still resulting in the same effect